rust_src/src/indent.rs

```diff
@@ -74,9 +74,9 @@ pub fn current_column() -> EmacsInt {
 ///
 /// The return value is the current column.
 #[lisp_fn(min = "1", intspec = "NMove to column")]
-pub fn move_to_column(column: EmacsInt, force: bool) -> EmacsInt {
+pub fn move_to_column(column: EmacsInt, force: LispObject) -> EmacsInt {
     column.check_natnum();
-    let buffer = &mut ThreadState::current_buffer();
+    let buffer = &mut ThreadState::current_buffer().unwrap();
```

Comment:
Use the `unchecked` variant instead of manually unwrapping.

Reply:
Fixed

Reply:
Did it revert?